clusters/build-clusters/01_cluster/machine-config/kernel_args_machineconfig.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: debug
+  name: write-queues-debug
+spec:
+  kernelArguments:
+    - 'nvme.write_queues=1'
```

Comment:
How do we check the current value of this on the node?

Reply:
You can do `dmesg | grep nvme` and it will give you the write_queues, should be 2 right now.

Reply:
> You can do `dmesg | grep nvme` and it will give you the write_queues, should be 2 right now.

```console
oc --context build01 debug node/ip-10-0-163-100.ec2.internal --as system:admin
Starting pod/ip-10-0-163-100ec2internal-debug ...
To use host binaries, run `chroot /host`
Pod IP: 10.0.163.100
If you don't see a command prompt, try pressing enter.
sh-4.4# chroot /host
sh-4.4# dmesg | grep nvme
sh-4.4#
```
Did I do it correclty?
Nothing returned.